### ISSUE TITLE
NPM Version crashes on Chome a11y audit failure

### ISSUE THREAD

**Issue by drewterry** [angular/protractor-accessibility-plugin#31]
The npm version of this tool doesn't include [PR#24](https://github.com/angular/protractor-accessibility-plugin/pull/24), and crashes on getOuterHTML when an audit fails.

**Comment by onishiweb** [comment]
Hi @sjelin @contolini @evilaliv3 do any of you have access or know anyone that does that could update the NPM version and resolve this issue please?
